File: Python/array/max_consecutive_ones.py
def findMaxConsecutiveOnes(nums: list) -> int:
    count, result = 0, 0
    for num in nums:
        if num == 1:
            count += 1
        else:
            result = max(result, count)
            count = 0
    
    return max(result, count)

if __name__ == "__main__":
    print(findMaxConsecutiveOnes([1,1,0,1,1,1]))

File: src/tmp/Java/src/com/dbc/algorithm/Array/MaxConsecutiveOnes.java
package com.dbc.algorithm.Array;

public class MaxConsecutiveOnes {
    public static int findMaxConsecutiveOnes(int[] nums) {
        int result = 0, count = 0;
        for (int num : nums) {
            if (num == 1) {
                count++;
            } else {
                result = Math.max(result, count);
                count = 0;
            }
        }

        return Math.max(result, count);
    }
}
